app/helpers/filters_helper.rb

```diff
@@ -56,6 +59,11 @@ def reset_filters_link(filter_type)
     end
   end
 
+  def managing_organisation_filter_options(user)
+    organisation_options = user.support? ? Organisation.all : [user.organisation] + user.organisation.managing_agents
```

Comment:
Don't have to change, but I think this might work `[user.organisation, *user.organisation.managing_agents]`